src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -183,20 +183,28 @@ public function getCookieJar()
     /**
      * Returns the current Crawler instance.
      *
-     * @return Crawler|null A Crawler instance
```

Comment:
should be reverted as it's still valid

Reply:
`null` is returned only for BC. There are no cases when `null` might be returned.

Reply:
fact is it's `Crawler|null` until 5.0

to me this is different then switching to new API for parameters (`@param`), we where want users to provide correct input beforehand.

Reply:
> fact is it's Crawler|null until 5.0

we don't document deprecated APIs

Reply:
:+1: sounds fair while thinking about it. Deprecation message should tell either way.